src/platform/elements/value/Value.ts

```diff
@@ -16,6 +16,10 @@ export enum NOVO_VALUE_THEME { DEFAULT, MOBILE };
                 <novo-entity-list *ngSwitchCase="NOVO_VALUE_TYPE.ENTITY_LIST" [data]='data' [meta]="meta"></novo-entity-list>
             </div>
 
+            <div *ngSwitchCase="NOVO_VALUE_TYPE.HTML" class="value-outer html-field">
+                <label>{{ meta.label }}</label>
```

Comment:
if the class is the only thing that is different for `HTML` then we don't need a switchcase within the template - we can use some kind of variable to retrieve class value and use it in the default case

Reply:
done